src/indent.cpp

```diff
@@ -4707,7 +4708,8 @@ void indent_preproc()
       }
       Chunk *next = pc->GetNextNcNnl();
 
-      if (next->IsNullChunk())
+      if (  pc != nullptr
+         && pc->IsNullChunk())
```

Comment:
Wrong replacement (```next``` to ```pc```) and not necessary because ```pc->GetNextNcNnl();``` will never return a ```nullptr``` by design.
This part of code should remain unchanged.

Reply:
Need to use ```next```, not ```pc```